packages/dotcom-ui-flags/src/server/formatFlagsJSON.ts

```diff
@@ -11,5 +11,5 @@ export default function formatFlagsJSON(flags: TFlagsData = {}): string {
     }
   })
 
-  return JSON.stringify(output)
+  return output
```

Comment:
Breaking change?

Reply:
Hmm. Both flags and appContext already seem to be in object form (rather than a stringified object), which is surprising.

This is a screen grab from the front page:
<img width="1077" alt="Screenshot 2020-05-05 at 12 58 24" src="https://user-images.githubusercontent.com/10484515/81063795-66f69800-8ed0-11ea-93d9-ff7dcbc045b3.png">